ExperimentDatabase.py

```diff
@@ -6,38 +6,34 @@ def __init__(self, file=":memory:"):  #call with file name as argument or no arg
         self._c = self._conn.cursor()
         try:
             self._c.execute('''CREATE TABLE experiment (
-                                experiment_id INTEGER PRIMARY KEY,
```

Comment:
Why are we removing experiment_id?

Reply:
We are not using experiment_id for anything. We are creating a different file for each experiment.